ext/reflection/php_reflection.c

```diff
@@ -2630,6 +2661,46 @@ ZEND_METHOD(reflection_parameter, getClass)
 }
 /* }}} */
 
+/* {{{ proto public bool ReflectionParameter::hasType()
+   Rethern whether parameter has a type hint */
```

Comment:
nit "Rethern", same for `hasReturnType()`.

(Also for Levi's peace of mind maybe drop the "hint" ^^)